circle.yml

```diff
@@ -33,10 +36,10 @@ deployment:
     commands:
       - NODE_ENV=production npm run deploy
       - cf login -a https://api.cloud.gov -u $CF_GSA_VOTE_USER -p $CF_GSA_VOTE_PASS -o gsa-ffd -s vote-production
-      - cf push
+      - cf zero-downtime-push vote-gov -f manifest.yml
   staging:
     branch: [staging]
     commands:
       - npm run deploy
       - cf login -a https://api.cloud.gov -u $CF_GSA_VOTE_USER -p $CF_GSA_VOTE_PASS -o gsa-ffd -s vote-staging
-      - cf push -f manifest-staging.yml
+      - cf zero-downtime-push vote-gov-staging -f manifest-staging.yml
```

Comment:
Awesome. I recommend multiple manifests and not inheriting from any of them as it's not a lot of stuff that needs to be changed. But either way, having multiple manifests is 💯